spec/memory-safe-d.dd

```diff
@@ -68,6 +68,9 @@ $(H2 $(LNAME2 usage, Usage))
          $(P These attributes may appear after the formal parameter list, in which case they apply either to a method's `this` parameter, or to
          a free function's first parameter $(I iff) it is `ref`.
          `return` or `scope` is ignored when applied to a type that is not a low-level pointer.)
+         
+         $(P $(B Note:) Checks for `scope` parameters are currently enabled
```

Comment:
I think `return` falls in the same category as well. Part of the checking introduced by `-dip25` covers it, but I think there are some additional checks with `-dip1000` as well.

Reply:
It is a bit awkward since return checks work even if the function is not @safe when -dip1000 is passed

Reply:
Yeah, it's confusing :/